docs/architecture/release_note_0_9.md

```diff
@@ -1,49 +0,0 @@
-# MXNet 0.9 (NNVM) Release Note
```

Comment:
We don't need those?

Reply:
All of the release notes are on github in the releases section. We don't maintain any others here. The presence of the page throws an error because it isn't in a ToC. I'd just delete it since the release notes are elsewhere.